src/main.rs

```diff
@@ -189,9 +198,9 @@ fn main() {
 
     let mut game_file_loader = GameFileLoader::default();
 
-    game_file_loader.add_archive("data.grf".to_string());
-    game_file_loader.add_archive("rdata.grf".to_string());
-    game_file_loader.add_archive("korangar.grf".to_string());
+    for game_file_name in config_loader.game_files_list {
+        game_file_loader.add_archive(game_file_name.to_string());
+    }
```

Comment:
I like the idea of hiding this even more, so maybe doing something like
```rust
config_loader.add_archives(&mut game_file_loader);
```
That would also allow you to make the `game_file_list` private, which is generally desireable